### PR TITLE
Feat: Support for the group ping feature

### PR DESCRIPTION
**Pull request by hasanlatch007**
Added Enum for the settings options and added `GroupPingOptions` column for new feat

### CODE REVIEW

**Comment by hasanlatch007** [comment]
If this is good to go dont merge it till i delete the function michel added for the group YesTheoryUploaded

**Review by geisterfurz007 — CHANGES_REQUESTED** [review]
Schema changes LGTM!

Please generate the migration (`yarn prisma migrate dev`) and bump the package version (`npm version minor`) and commit both onto this branch as well :)

File: prisma/schema.prisma [hunk 1]
```diff
@@ -92,10 +92,18 @@ model UserGroup {
   lastUsed                        DateTime                          @default(dbgenerated("'1969-12-31 23:00:00'::timestamp without time zone")) @db.Timestamp(6) @map("last_used")
   cooldown                        Int                               @default(60)
   deadtime                        Int                               @default(0)
+  groupPingOption                 GroupPingSetting                  @default(MEMBER)
```
Comment (geisterfurz007) [inline prisma/schema.prisma:95]:
```suggestion
  groupPingSetting                GroupPingSetting                  @default(MEMBER)
```

**Review by geisterfurz007 — CHANGES_REQUESTED** [review]

**Comment by hasanlatch007** [comment]
Ignore the multiple version commits my console lagged and it was doing nothing and suddenly all synced the current one is the last commit

**Review by geisterfurz007 — APPROVED** [review]

**Pull request merged**